README.md

```diff
@@ -1 +1,50 @@
-# deployment-deploy-databases
\ No newline at end of file
+Deploy new MySQL database using our playbooks
+=============================================
+
+This ansible repository deploys mysql server on OpenStack provider. 
+
+Create a OpenStack environment
+------------------------------
+
+1. Create 10GB volume for root filesystem, create it using image "Ubuntu 14.04". 
+2. Create volume for data, as big as you need. It should be empty.
+3. Create a security group for database servers it should have following rules:
+    1. Incoming traffic is allowed only for port 22
+    2. Outgoing traffic should be allowed. 
+    3. When you provision servers using the database allow them using their IP.
+3. After volume for root fs is ready create a VM for the mysql:
+    1. It should use root volume as filesystem (`Boot Source` -> `Volume` -> `Your root volume`)
```

Comment:
@antoviaque No. Mysql instance works mounted on two volumes, first in on `/dev/vda` and translates to `/`, second is on `/dev/vdb` and is mounted on `/var/lib/mysql`.